services/entitlement/service/src/index.ts

```diff
@@ -116,7 +116,10 @@ async function bootstrap(): Promise<void> {
 
   const server = app.listen(config.port, () => {
     if (config.isDev) {
-      logger.log(`http://localhost:${config.port}/graphiql`);
+      logger.log({
+        message: `🚀 Server ready at http://localhost:${config.port}`,
```

Comment:
I would still include `/graphiql`, either here or as a separate URL. It's
inconvenient to lose a way to Ctrl+click on URL in terminal to open the graphiql
UI right away.

Reply:
Same for catalog index.ts